pcbuilder_crawlers/templates/video_card.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": ["Productnaam"],
+	"subname": [],
+	"info": [],
+	"stock": [],
+	"image": [],
+	"ean": ["EAN code"],
+	"sku": [],
+	"SKU": [],
+	"link": [],
+	"brand": ["Merk"],
+	"execution": [],
+	"memory_specification": [],
+	"serie": [],
+	"product": [],
+	"memory_size": [],
+	"number": [],
+	"module_size": [],
+	"Price_per_gb": [],
+	"memory_type": [],
```

Comment:
So yea, sadly this is all a bit too much :( Let's make a list of fields we're going to validate on since those are the fields we want translated. We don't care what the others are called.

Also you probably want the same implementation as what you're doing in the category template. Where you check if it's in the value array and if so translate the key to the template key. This will require the key to be present in the value array.:
```
{
"ean": ["EAN code", "ean"]
}
```